MidasCivil_Adapter/Convert/ToMidasCivil/Elements/FromFEMesh.cs

```diff
@@ -24,18 +24,24 @@
 using BH.Engine.Adapter;
 using BH.oM.Structure.Elements;
 using System.Collections.Generic;
+using System;
+using BH.Adapter.MidasCivil;
+using System.Text;
+
+
 namespace BH.Adapter.Adapters.MidasCivil
 {
     public static partial class Convert
     {
         /***************************************************/
         /**** Public Methods                            ****/
         /***************************************************/
-
-        public static string FromFEMesh(this FEMesh feMesh)
+        private static int index = 0;
+        public static string FromFEMesh(this FEMesh feMesh, int index_ = 1)
         {
+            index = Math.Max(index, index_);
+            StringBuilder midasElements = new StringBuilder();
             string midasElement = "";
```

Comment:
This can be initialised within the `foreach` loop now that you're using the `StringBuilder`.